### DISCUSSION TITLE
Remove top and bottom background in sub-menu

### DISCUSSION THREAD

**Discussion by longkimkao** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3642] [Q&A]
Helo Keboo, how do i remove the red background on top and bottom of submenu ?
![2024-08-06_16-15-19](https://github.com/user-attachments/assets/aab9f021-5ce7-4b98-a2e9-801070e4b3bc)

**Comment by nicolaihenriksen** [comment]
@longkimkao An attached property (`MenuAssist.MenuItemsPresenterMargin`) has been added with PR #3760 to control this.
